packages/eslint-plugin/src/rules/shorthand-property-sorting/index.ts

```diff
@@ -0,0 +1,124 @@
+import type { Rule } from 'eslint';
+
+import { shorthandFor } from './utils';
+
+const findProgramRoot = (node: Rule.Node) => {
```

Comment:
ah looks like this type error is because typescript version in compiled is outdated, let me fix that

Reply:
never mind, it's a bit of a pain to upgrade typescript 😓